README.md

```diff
@@ -16,24 +16,24 @@ Karavi Topology is part of the Karavi open source suite of Kubernetes storage en
 [![License](https://img.shields.io/github/license/dell/karavi-topology)](LICENSE)
 [![Docker Pulls](https://img.shields.io/docker/pulls/dellemc/karavi-topology)](https://hub.docker.com/r/dellemc/karavi-topology)
 [![Go version](https://img.shields.io/github/go-mod/go-version/dell/karavi-topology)](go.mod)
-[![Latest Release](https://img.shields.io/github/v/release/dell/karavi-topology?label=latest&style=flat-square)](https://github.com/dell/karavi-topology/releases)
+[![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/dell/karavi-topology?include_prereleases&label=latest&style=flat-square)](https://github.com/dell/karavi-topology/releases/latest)
 
-This project provides visibility into Dell EMC CSI (Container Storage Interface) driver provisioned volume characteristics in Kubernetes correlated with volumes on the storage system.  
+This project provides visibility into Dell EMC CSI (Container Storage Interface) driver provisioned volume characteristics in Kubernetes correlated with volumes on the storage system.
```

Comment:
We want to be as consistent with the upper level Observability umbrella here and bring together the description in a simple manner for our end users. Below we have "This project provides Kubernetes administrators with the topology data related to containerized storage. This topology data is visualized using Grafana."  under the Observability umbrella we have "Observability provides standardized approaches for storage observability." and here we have "This project provides visibility into Dell EMC CSI (Container Storage Interface) driver provisioned volume characteristics in Kubernetes correlated with volumes on the storage system." Let's find a single simple statement that captures the essence of the project. Thanks!

Reply:
Thanks for your feedback @AronAtDell.  Since this isn't related to the issue being fixed in this PR, can you kindly create a new bug report issue.  Thank you!

Reply:
@gallacher, @medegw01  - I thought that the Team was addressing documentation changes as well seeing that the fix was limited to line 19 and we are making documentation changes throughout this PR, which is why I added this comment.